g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/RsaCryptUtils.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.l10n.conf;
+ 
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+/**
+ * 
+ * it use to decode the access key and Secret key
+ *
+ */
+public class RsaCryptUtils {
```

Comment:
I recommend change a other name for this class. because the same class name have defined in i18n manager

Reply:
Done